components/resources/decentralized_dns_strings.grdp

```diff
@@ -40,16 +40,16 @@
     Enable support of Solana Name Service (SNS) in Brave?
   </message>
   <message name="IDS_SNS_OPT_IN_PRIMARY_PARAGRAPH" desc="Main paragraph of SNS opt-in interstitial page.">
-    Brave will be using Syndica to resolve .sol domain names. Brave hides your IP address. If you enable this, Syndica will see that someone is trying to visit these .sol domains but nothing else. See Syndica's <ph name="BEGIN_SYNDICA_TOU_LINK">&lt;a target="_blank" href="$1"&gt;</ph>terms of use<ph name="END_SYNDICA_TOU_LINK">&lt;/a&gt;</ph> and <ph name="BEGIN_SYNDICA_PRIVACY_LINK">&lt;a target="_blank" href="$2"&gt;</ph>privacy policy<ph name="END_SYNDICA_PRIVACY_LINK">&lt;/a&gt;</ph>.
+    Brave will be using a third-party to resolve .sol domain names. Brave hides your IP address. If you enable this, the third-party will see that someone is trying to visit these .sol domains but nothing else. For more information about which third-parties we use and their privacy policies, please see our <ph name="BEGIN_SNS_WIKI_LINK">&lt;a target="_blank" href="$1"&gt;</ph>help page<ph name="END_SNS_WIKI_LINK">&lt;/a&gt;</ph>.
   </message>
   <message name="IDS_DECENTRALIZED_DNS_OPT_IN_PRIMARY_BUTTON" desc="The text for the decentralized DNS opt-in interstitial primary button.">
     Proceed using Infura server
   </message>
   <message name="IDS_DECENTRALIZED_DNS_OPT_IN_PRIMARY_INFURA_BUTTON" desc="The text for the decentralized DNS opt-in interstitial primary button for Infura.">
     Proceed using Infura server
   </message>
-  <message name="IDS_DECENTRALIZED_DNS_OPT_IN_PRIMARY_SYNDICA_BUTTON" desc="The text for the decentralized DNS opt-in interstitial primary button for Syndica.">
-    Proceed using Syndica server
+  <message name="IDS_DECENTRALIZED_DNS_OPT_IN_PRIMARY_SNS_BUTTON" desc="The text for the decentralized DNS opt-in interstitial primary button for SNS server.">
+    Proceed using an SNS server
```

Comment:
Maybe `Proceed using a third-party SNS server` or `Proceed using a third-party server` ?
So it matches replacement of `Syndica` with `third-party` above

Reply:
https://github.com/brave/devops/issues/10126#issuecomment-1850182954 was just using the exact wording here. :)